typespec-extension/package.json

```diff
@@ -44,15 +44,20 @@
     "!dist/test/**",
     "target/azure-typespec-extension-jar-with-dependencies.jar"
   ],
-  "peerDependencies": {
-    "@typespec/compiler": "~0.42.0",
-    "@typespec/rest": "~0.42.0",
-    "@typespec/versioning": "~0.42.0",
-    "@azure-tools/typespec-azure-core": "~0.28.0",
-    "@azure-tools/typespec-client-generator-core": "~0.28.0"
-  },
   "dependencies": {
     "@autorest/codemodel": "~4.19.2",
+    "@autorest/java": "4.1.16",
```

Comment:
why do we add autorest/java? (I don't think we publish autorest/java to npm, please correct me if I am wrong.)

Reply:
I think the recommendation from Wes (maybe the outdated), is not adding unneeded dependency into the typespec-java lib.

We aware .NET has it, but it seems yet to be an agreement across emitters (I think python does not do this).

But anyway, not autorest/java. It maybe typespec-autorest etc.